tests/regression/roles/sonic_vlans/defaults/main.yml

```
@@ -38,54 +43,62 @@ tests:
       - vlan_id: 500
         description: "modified vlan500 description"
       - vlan_id: 501
+      - vlan_id: 503
+      # Odd case where delete sets things to default true
+        autostate: true
   # replace test cases started
   - name: test_case_04
     description: Replace VLANs
     state: replaced
     input:
-        - vlan_id: 500
-        - vlan_id: 502
-          description: "modified vlan502 description"
+      - vlan_id: 500
+        autostate: true
+      - vlan_id: 502
+        description: "modified vlan502 description"
+      - vlan_id: 506
   # overridden test cases started
   - name: test_case_05
     description: Override VLANs configuration
     state: overridden
     input:
-        - vlan_id: 501
-        - vlan_id: 502
-          description: "overridden vlan502 description"
-        - vlan_id: 504
+      - vlan_id: 501
+      - vlan_id: 502
+        description: "overridden vlan502 description"
+        autostate: true
+      - vlan_id: 504
```

It would be good to verify that setting (chaning) 'autostate' to 'false' works correctly in 'overridden' state.

```suggestion
      - vlan_id: 501
      - vlan_id: 502
        description: "overridden vlan502 description"
        autostate: true
      - vlan_id: 503
        autostate: false
      - vlan_id: 504
```